skipruntime-ts/ts/examples/utils.ts

```diff
@@ -66,7 +66,7 @@ class Player {
 
   start(idx: number) {
     const scenario = this.scenarios[idx - 1];
-    if (!scenario) {
+    if (scenario.length == 0) {
```

Comment:
Hmm, not sure about this change. Don't we want to check `scenario !== undefined`?

Reply:
Although the linter thinks that `scenario !== undefined` is always false.

Reply:
That's because we disabled `noUncheckedIndexedAccess` in `tsconfig.json`. It was too noisy. Ideally we'll re-enable it

Reply:
So since we don't want `noUncheckedIndexedAccess` to be reported everywhere, it complains if we ever do check the index access. FFS.